# Status of an AIRSKIN consisting of several pads

float32[] pressures